package.json

```diff
@@ -14,17 +15,19 @@
     "babel-polyfill": "^6.16.0",
     "babel-preset-latest": "6.22.0",
     "concurrently": "^3.1.0",
-    "nodemon": "^1.11.0",
-    "jest": "18.1.0"
+    "jest": "18.1.0",
+    "nodemon": "^1.11.0"
+  },
+  "engines": {
+    "node": ">= 6.0"
   },
-  "engines": { "node": ">= 6.0" },
   "main": "lib/index.js",
   "name": "teleport.js",
   "scripts": {
     "compile": "babel --presets latest -d lib/ src/",
     "postinstall": "rm -rf lib && mkdir -p lib && npm run compile",
     "link": "source bin/link.sh",
-    "test": "jest",
+    "test": "export TEST_RANDOM=$RANDOM && jest tests/create.test.js && jest tests/utility_methods.test.js && jest tests/deploy_methods.test.js",
```

Comment:
jest by default picks up `test.js` files, so you don't need to specify them like this.